templates/typescript_samples/samples/generated/$version/_util.njk

```diff
@@ -25,6 +28,13 @@
 {%- endfor -%}
 {%- endmacro -%}
 
+{%- macro markDownStuff(formattedLine) %}
```

Comment:
Could you rename `markDownStuff` to some name readable? 😆 
And looks like this method has not been used.